common/src/main/java/com/wynntils/models/items/annotators/game/UnknownGearAnnotator.java

```diff
@@ -18,15 +19,15 @@ public final class UnknownGearAnnotator implements ItemAnnotator {
     private static final Pattern UNKNOWN_GEAR_PATTERN = Pattern.compile("^§[5abcdef](.*)$");
 
     @Override
-    public ItemAnnotation getAnnotation(ItemStack itemStack, CodedString name) {
+    public ItemAnnotation getAnnotation(ItemStack itemStack, StyledText name) {
         GearType gearType = GearType.fromItemStack(itemStack);
         if (gearType == null) return null;
 
         Matcher matcher = name.getMatcher(UNKNOWN_GEAR_PATTERN);
         if (!matcher.matches()) return null;
 
         String gearName = matcher.group(1);
-        GearTier gearTier = GearTier.fromFormattedString(name);
+        GearTier gearTier = GearTier.fromFormattedString(CodedString.fromStyledText(name));
```

Comment:
This appears multiple times. It might be that basically all calls to this function is from annotators. You might want to check if it is possibly to convert it to use StyledText instead. (Even if you only put the odedString.fromStyledText(name)) call inside fromFormattedString, it's an improvement.

Reply:
If so, also please rename it, to `fromStyledText` or similar.